### ISSUE TITLE
Upgrade 2022-05-02 / Message in screen

### ISSUE THREAD

**Issue by fly-man-** [hakwerk/labca#45]
```
[AUDIT] timed out waiting for ca2.boulder:9093 health check 21 seconds
[AUDIT] timed out waiting for ra2.boulder:9094 health check 57 seconds
[AUDIT] Couldn't load rate limit policies file: yaml: line 19: could not find expected ':' 1 minute 
```

After the labca upgrade, this message set keeps repeating every 50 seconds.

Were there any changes that could cause this ??

Output of **docker ps -a**
```
CONTAINER ID   IMAGE                                           COMMAND                  CREATED          STATUS                                  PORTS                                                                      NAMES
62ec168df593   letsencrypt/boulder-tools:go1.17.9_2022-04-12   "labca/entrypoint.sh"    12 minutes ago   Restarting (1) Less than a second ago                                                                              boulder-boulder-1
b848bd0aea0a   letsencrypt/boulder-tools:go1.17.9_2022-04-12   "./setup.sh"             12 minutes ago   Up 12 minutes                           3000/tcp                                                                   boulder-labca-1
d73cbcffeb47   mariadb:10.5                                    "docker-entrypoint.s…"   12 minutes ago   Up 12 minutes                           3306/tcp                                                                   boulder-bmysql-1
8f6816c53f4c   letsencrypt/boulder-tools:go1.17.9_2022-04-12   "./control.sh"           12 minutes ago   Up 4 minutes                            3030/tcp                                                                   boulder-control-1
602bcaf7648c   nginx:1.21.6                                    "/docker-entrypoint.…"   12 minutes ago   Up 12 minutes                           0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp   boulder-nginx-1
```

**Comment by hakwerk** [comment]
Looks like your `/home/labca/boulder_labca/rate-limit-policies.yml` file got corrupted during the upgrade.

Possibly [this commit](https://github.com/hakwerk/labca/commit/169b1470781dde0bbe23f1a5091e2302a56ad041) is the culprit then, where I try to add the rateLimitsURL to that config file.

If you can see anything obviously wrong in your `rate-limit-policies.yml` then maybe you can try to fix it so the application will at least run again.
I'll have a closer look when I get a chance

**Comment by hakwerk** [comment]
I have fixed the issue with the multiple domains again messing up the config generation. This should be fixed in latest release v22.05.1

**Comment by fly-man-** [comment]
Tested the latest version, error is done

Thanks for the quick fix
